### ISSUE TITLE
Packager for iOS not including levure.livecodescript

### ISSUE THREAD

**Issue by spencerlearning** [trevordevore/levure#35]
I built for ios using tdk-mobile-packager and it worked well except it did not appear to include `levure.livecodescript` in the bundle. The result was the standalone loaded and ran on my iPad but the application did not load. Here are the contents of the bundle built from the starter app:

![tdk-mobile-packager-1](https://cloud.githubusercontent.com/assets/1884922/24788741/f7f4f8fc-1b23-11e7-917c-afdd9666b7b8.png)

The good news is the bundle was built without errors, the bundle loaded and ran on an iPad via Xcode, and the `helpers` and `ui` folders in the bundle have the correct contents. The missing `levure.livecodescript` seems like the only problem.

1. Follow the wiki instructions to [create a new project from master](https://github.com/trevordevore/levure/wiki/Set-Up-Your-Project-Folder).

2. Update `levure.livecodescript` and `packager.livecodescript` with the versions from tdk-mobile-packager.

3. Open `standalone.livecode` in a new instance of Livecode.

3. In Standalone Application Settings accept defaults except iOS is the only platform selected.

4. In message box, run `levurePackageApplication "beta"`. It builds successfully!

5. Drag the ios build to Xcode Devices and then run it on iPad.

**Comment by spencerlearning** [comment]
Android seems to have a similar problem: `levure.livecodescript` is not in the build. Windows and Mac do not have the problem.

**Comment by trevordevore** [comment]
Looks like the mobile standalone builder doesn't pick up the behavior. I just pushed some updates. Can you try again and let me know how it goes?

**Comment by spencerlearning** [comment]
It runs the starter app on my iPad! I built for ios with no inclusions checked...success. I built with ALL cross-platform (Mac, Windows, Linux, iOS, and Android) inclusions checked except for Geometry...success.

The only error I got was with the Geometry inclusion. The build was successful but when I ran it on the iPad it gave an error dialog:

```
219,29,21,REVTargetStack
81,29,21
198,29,21
252,29,21
252,29,21
241,29,1,resizeStack
353,0,0,stack "revgeometrylibrary"
```

Since none of the other inclusions generated an error, this seems to be a Geometry problem, not a Levure problem.

I'll test on Android next and post the results shortly.
